### DISCUSSION TITLE
What is the relationship between the different version numbering method?

### DISCUSSION THREAD

**Discussion by yaelperl1234** [dask/s3fs#708] [Q&A] [answered]
major.minor.patch vs year._._

In my python env s3fs==0.4.2, how can I find the matched release?

**Comment by ianthomas23** [comment, marked as answer]
There are a number of places you can check release information:

- https://s3fs.readthedocs.io/en/latest/changelog.html
- https://github.com/fsspec/s3fs/tags
- https://pypi.org/project/s3fs/#history

`s3fs` switched from SemVer (e.g. 0.4.2) to CalVer (e.g. 2023.3.0) in April 2021. Version 0.4.2 was released at the end of March 2020.
